solr/CHANGES.txt

```diff
@@ -67,6 +67,8 @@ Deprecation Removals
 
 * SOLR-17284: Remove deprecated BlobRepository in favour of FileStore. (Eric Pugh)
 
+* SOLR-17313: Remove deprecated class SolrLogPostTool.  PostLogsTool and the bin/solr post command has replaced this. (Eric Pugh)
```

Comment:
I argue we needn't cluster CHANGES.txt with a change of this nature.  Nobody will care about this.  No feature was removed; it was an internal change of where the functionality lives.